android/features/music-player/src/main/java/io/newm/feature/musicplayer/MusicPlayerViewer.kt

```diff
@@ -103,6 +103,7 @@ internal fun MusicPlayerViewer(
                 modifier = Modifier.align(Alignment.Center),
                 model = ImageRequest.Builder(context)
                     .data(song.artworkUri)
```

Comment:
Did not add a placeholder here, because it shows the placeholder every time. Not ideal for the experience we have on the media player.